content/sensu-go/6.1/plugins/supported-integrations/influxdb.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "InfluxDB integration"
+linkTitle: "InfluxDB"
+description: "Use the Sensu InfluxDB Handler plugin to integrate Sensu with your existing InfluxDB workflows. Read about the features of Sensu's InfluxDB integration and learn how to get the plugin."
+version: "6.1"
+product: "Sensu Go"
+menu: 
+  sensu-go-6.1:
+    parent: supported-integrations
+---
+
+The Sensu InfluxDB Handler plugin is a Sensu [handler][1] that sends Sensu metrics to the time-series database InfluxDB so you can store, instrument, and visualize Sensu metrics data.
+You can also use the Sensu InfluxDB Handler integration to create metrics from Sensu status check results for long-term storage in InfluxDB.
+
+{{% notice note %}}
+**NOTE**: The Sensu InfluxDB Handler plugin is an example of Sensu's status and metrics processing and storage integrations.
+To find more integrations, search [Bonsai, the Sensu asset index](https://bonsai.sensu.io/).
+{{% /notice %}}
+
+## Features
+
+- Transform metrics to InfluxDB format: extract and transform the metrics you collect from different sources like Graphite, OpenTSDB, Nagios, and Influx and populate them into InfluxDB.
```

Comment:
```suggestion
- Transform metrics to InfluxDB format: extract and transform the metrics you collect from different sources in formats like Graphite, OpenTSDB, Nagios, and Influx and populate them into InfluxDB.
```

Reply:
This phrasing should be updated in pages for other integrations in "Status and metrics processing and storage"